src_py/__init__.py

```diff
@@ -100,61 +106,97 @@ def warn(self):
     try:
         import pygame.cdrom
     except (ImportError, IOError):
+        # This creates a missing module object
```

Comment:
Generally we only need to comment one time to make something clear.

Reply:
Adding a docstring to the MissingModule class would make this more clear.

Also this comment could probably be improved:
```
# next, the "standard" modules
# we still allow them to be missing for stripped down pygame distributions
```

- Sentences with capital letters and full stops.
- Perhaps it could talk about how missing modules are assigned the MissingModule class instance. Because that's a bit of a weird thing to do compared to other python modules.